roles/ztp/setup_cluster_image_set/README.md

```diff
@@ -0,0 +1,27 @@
+# Setup Cluster Image Set
+
+This role clones the ZTP GitOps repository, reads the clusterImageSetRef fields in the site config manifests and creates the required clusterImageSet objects in the hub cluster for the spoke cluster deployment to work.
+
+## Variables
+
+Variable               | Default | Required | Description
+-----------------------|---------|----------|-------------
+scis_repo_url          |         | yes      | URL to the repository with the GitOps manifests.
+scis_sites_path        |         | yes      | Path in the repository to the directory containing the site config manifests.
+scis_branch            | main    | no       | Branch to clone.
+scis_username          |         | yes      | Username to log into the repository.
```

Comment:
Should we include the option to access to the repo through key_path, apart from enabling username-password?

Reply:
You're right, we should add support for SSH keys.